apps/user-profile/src/pages/users/[userId].js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import axios from "axios";
+
+import UserProfile from "../../components/modules/UserProfile";
+
+export const getStaticPaths = async () => {
+  return {
+    paths: [{ params: { userId: "/"}}],
+    fallback: "blocking",
```

Comment:
@judeamos why do you have empty paths?

Reply:
I wanted next.js to allow users to enter any value in the `users/[userId]` URL, but I couldn't set userId to be an empty string `userId: ""`, it was throwing a compilation error.  Adding an empty path fixed it. 
 
I've just learnt that i could pass an empty array as paths `paths: []`, so I've updated the code to now pass an empty array, instead of `[{params: {userId: "/"}}]`.